### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Package RPM
+-----------
+
+Named in the form ``<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id>`` a package RPM represents the conda "package cache" (the thing that normally lives in `<root prefix>/.pkgs/<pkg name>-<pkg version>-<pkg build id>`). A package RPM *does not* express its dependencies and can not be usefully installed as a standalone entity.
+
+Tagged environment RPM
+----------------------
+
+Named in the form ``<RPM prefix>-env-<env name>-tag-<env tag>`` a tagged environment RPM represents a resolved conda environment.
+It depends on all Package RPMs that should be installed in order to produce a working environment. The tagged environment RPM knows it target installation prefix, and uses conda functionality at install time to link the Package RPMs to the desired installation prefix.
```
Comment (lbdreyer) [inline README.md:24]: > The tagged environment RPM knows it target installation prefix

typo? 

(Well explained btw)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+Create RPMs from a conda-gitenv
+===============================
+
+conda-rpms is designed to convert a resolved [conda-gitenv](https://github.com/SciTools/conda-gitenv) environment into a collection of RPM specs suitable for deployment on compatible platforms (Red Hat Enterprise Linux, Fedora, etc.).
+
+A high level description of the motivation for the creation of RPMs from a [conda-gitenv](https://github.com/SciTools/conda-gitenv) is described in the [centrally managed scientific software workflow](https://rawgit.com/pelson/conda-rpms/master/tmp_overview_docs/scitools-env-workflow.html) document.
+
+The code to do the conda distribution to RPM spec transformation itself is relatively simple, drawing on tools such as gitpython, Jinja2 and conda.
```
Comment (lbdreyer) [inline README.md:8]: I suspect the purpose of this line is to inform people who are familiar with the area how conda-rpms works.

I would reword it, to something along the lines:
To transform the conda distribution to RPM specs, conda-rpms uses tools including git-python, Jinja, conda

I don't like the inclusion of "relatively simple".

File: README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Because a built RPM knows its destined installation location, a number of RPM abstractions have been made that enable us to retain conda's hard-linking and relocatability benefits.
+
+RPM Types
+=========
+
+Package RPM
+-----------
+
+Named in the form ``<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id>`` a package RPM represents the conda "package cache" (the thing that normally lives in `<root prefix>/.pkgs/<pkg name>-<pkg version>-<pkg build id>`). A package RPM *does not* express its dependencies and can not be usefully installed as a standalone entity.
```
Comment (lbdreyer) [inline README.md:18]: It may be clearer to break out the naming from the explanation as the name is quite long and not enormously readable if you have no prior context.
Possibly separate it onto a new line or even put it at the end.

File: README.md (hunk 1 above)
Comment (pelson) [inline README.md:24]: Thank you. I can't see the typo, can you help a poor speller :wink:?

Comment (pelson) [inline README.md:24]: _its_.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Named in the form ``<RPM prefix>-env-<env name>-tag-<env tag>`` a tagged environment RPM represents a resolved conda environment.
+It depends on all Package RPMs that should be installed in order to produce a working environment. The tagged environment RPM knows it target installation prefix, and uses conda functionality at install time to link the Package RPMs to the desired installation prefix.
+
+Labelled environment RPM
+------------------------
+
+Named in the form `<RPM prefix>-env-<env name>-label-<label>`, a labelled environment is simply a meta package which depends on one, and only one, Tagged Environment RPM. Unlike the immutable tagged environment, a labelled environment will tend to change to reference newer tags as time moves on. This change is reflected in the labelled environment's `version`, meaning that it is only possible to ever have one labelled environment RPM installed per environment name.
+
+
+Usage
```
Comment (lbdreyer) [inline README.md:32]: I wonder whether this section should be above the RPM Types section. 

The RPM Types is detail, whereas the usage section is smaller and a bit more high level.

You don't necessarily need to understand what each of the RPMs are before you use it. You could always build your specs, then read more to figure out what each thing you've produced is.

File: README.md (hunk 3 above)
Comment (pelson) [inline README.md:18]: To help, would you mind making a suggestion?

**Comment by lbdreyer** [comment]
I have made a few comments, that are very much intended as suggestions and I will completely understand if you disagree with them - everyone reads and takes in information differently so docs can be difficult to get right.

Personally, I just find when I read documentation of a new thing, I'm a bit impatient and want to be able to understand what is going onasap. I don't want to lose the interest of potential users by putting too much detail too soon.

File: README.md (hunk 3 above)
Comment (lbdreyer) [inline README.md:18]: ## Package RPM

RPM name format:`<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id>`

A package RPM represents the conda "package cache" (the thing that normally lives in `<root prefix>/.pkgs/<pkg name>-<pkg version>-<pkg build id>`). A package RPM _does not_ express its dependencies and can not be usefully installed as a standalone entity.

Comment (lbdreyer) [inline README.md:18]: I'm not sure about 'RPM name format' 

Maybe some special formatting?
**RPM name format:**`<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id>`

I just want it to be clear that that section isn't a part of the explanation of what a package rpm is.

**Comment by pelson** [comment]
Thanks @lbdreyer. I followed this up with https://github.com/pelson/conda-rpms/pull/25.
